nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/GetHBase.java

```diff
@@ -147,14 +146,6 @@
             .allowableValues(NONE, CURRENT_TIME)
             .defaultValue(NONE.getValue())
             .build();
-    static final PropertyDescriptor AUTHORIZATIONS = new PropertyDescriptor.Builder()
```

Comment:
Is this an unused property? I see it being removed but didn't see it moved anywhere

Reply:
VisibilityFetchSupport provides a version of it that is fungible.